tests/integrations/tests/test_google.py

```
@@ -882,6 +886,172 @@ def test_25_speech_generation_language_support(self, google_client, test_config,
+        ]
+        
+        keyword_matches = sum(
+            1 for keyword in reasoning_keywords if keyword in response_lower
+        )
+        assert keyword_matches >= 3, (
+            f"Response should address the farmer problem. "
+            f"Found {keyword_matches} keywords. Content: {regular_text[:200]}..."
+        )
+
+    @skip_if_no_api_key("google")
+    def test_27_extended_thinking_streaming(self, google_client, test_config):
```

Ruff (0.14.8)

Unused method argument: `test_config`

(ARG002)